server.js

```diff
@@ -1,30 +1,79 @@
-import express from "express";
-import cors from "cors";
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
-const port = process.env.PORT || 8080;
-const app = express();
+import express from 'express'
+import cors from 'cors'
+import expressListEndpoints from 'express-list-endpoints'
+import artistData from './data/artists.json'
+
+// Defines the port the app will run on
+const port = process.env.PORT || 8080
+const app = express()
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors());
-app.use(express.json());
+app.use(cors())
+app.use(express.json())
 
 // Start defining your routes here
-app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
-});
+app.get('/', (req, res) => {
+  const endpoints = expressListEndpoints(app)
+  res.json(endpoints)
+})
```

Comment:
would be good to add to the documentation how to filter data using the query `alive` like:

```javascript
app.get('/', (req, res) => {
  const endpoints = expressListEndpoints(app).map(endpoint => {
    // Add alive query parameter to the '/artists/:name' endpoint
    if (endpoint.path === '/artists/:name') {
      endpoint.query = [{
        parameter: "alive",
        description: "Filter artists by whether they are alive or deceased.",
        type: "boolean",
        example: "true"
      }];
    }
    return endpoint;
  });

  res.json(endpoints);
});
``` 

in this way we inform other devs about how to fully use the api! ⭐